src/dynamics/rigid_body_set.rs

```diff
@@ -23,7 +23,7 @@ impl BodyPair {
 }
 
 #[cfg_attr(feature = "serde-serialize", derive(Serialize, Deserialize))]
-#[derive(Clone, Default)]
+#[derive(Clone, Default, Debug)]
```

Comment:
irrelevant but 🤷 ; let me know if you prefer it out of the PR.

Reply:
That’s fine.